lib/galaxy/webapps/base/webapp.py

```diff
@@ -322,6 +322,12 @@ def __init__(
             # This is a web request, get or create session.
             assert session_cookie
             self._ensure_valid_session(session_cookie)
+
+        try:
+            self.app.authnz_manager.refresh_expiring_oidc_tokens(self)  # type: ignore[attr-defined]
+        except Exception:
+            pass
```

Comment:
Yes, I think that would be more explicit and not hide other real exceptions.

```suggestion
        if hasattr(trans.app, "authnz_manager"):
            self.app.authnz_manager.refresh_expiring_oidc_tokens(self)
```